### PR TITLE
feat: release APISIX 3.4.0

### PR DESCRIPTION
**Pull request by AlinsRan**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by lingsamuel — DISMISSED** [review]

File: docs/zh/latest/CHANGELOG.md [hunk 1]
```diff
@@ -70,6 +71,38 @@ title: CHANGELOG
...
+### Plugins
+
+- :sunrise: 支持 proxy_rewrite 插件的多个正则表达式匹配 [#9194](https://github.com/apache/apisix/pull/9194)
+- :sunrise: 添加 loki-logger 插件 [#9399](https://github.com/apache/apisix/pull/9399)
+- :sunrise: 允许用户为 prometheus 插件配置 DEFAULT_BUCKETS [#9673](https://github.com/apache/apisix/pull/9673)
+
+### Bugfixes
+
+- 修复 (body-transformer)：xml2lua 将空表替换为空字符串 [#9669](https://github.com/apache/apisix/pull/9669)
+- 修复：保存和恢复 pb 状态 [#9606](https://github.com/apache/apisix/pull/9606)
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:92]: @kingluo Please provide appropriate title.

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -70,6 +71,38 @@ title: CHANGELOG
...
+- 修复 (skywalking-logger, error-log-logger)：支持在 skywalking service_instance_name 中使用 $hostname [#9401](https://github.com/apache/apisix/pull/9401)
+- 修复 (admin)：secrets 不支持通过 PATCH 更新属性 (#9510)[#9510](https://github.com/apache/apisix/pull/9510)
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:94]:
```suggestion
- 修复 (admin)：修复 secrets 不支持通过 PATCH 更新属性 [#9510](https://github.com/apache/apisix/pull/9510)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -70,6 +71,38 @@ title: Changelog
...
+- :sunrise: Support miltiple regex pattern matching for proxy_rewrite plugin [#9194](https://github.com/apache/apisix/pull/9194)
+- :sunrise: Add loki-logger plugin [#9399](https://github.com/apache/apisix/pull/9399)
+- :sunrise: Allow user configure DEFAULT_BUCKETS for prometheus plugin  [#9673](https://github.com/apache/apisix/pull/9673)
+
+### Bugfixes
+
+- Fix(body-transformer): xml2lua: replace empty table with empty string [#9669](https://github.com/apache/apisix/pull/9669)
+- Fix: save and restore pb state [#9606](https://github.com/apache/apisix/pull/9606)
+- Fix(skywalking-logger, error-log-logger): support $hostname in skywalking service_instance_name [#9401](https://github.com/apache/apisix/pull/9401)
+- Fix(admin): secrets do not support to update attributes by PATCH (#9510)[#9510](https://github.com/apache/apisix/pull/9510)
```
Comment (leslie-tsang) [inline CHANGELOG.md:94]:
```suggestion
- Fix(admin): fix secrets do not support to update attributes by PATCH [#9510](https://github.com/apache/apisix/pull/9510)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -70,6 +71,38 @@ title: Changelog
...
+- :sunrise: Support route-level MTLS [#9332](https://github.com/apache/apisix/pull/9332)
+- :sunrise: Support id schema for global_rules [#9517](https://github.com/apache/apisix/pull/9517)
+- :sunrise: Support use a single long http connection to watch all resources for etcd [#9456](https://github.com/apache/apisix/pull/9456)
+- :sunrise: Support max len 256 for ssl label [#9301](https://github.com/apache/apisix/pull/9301)
+
+### Plugins
+
+- :sunrise: Support miltiple regex pattern matching for proxy_rewrite plugin [#9194](https://github.com/apache/apisix/pull/9194)
+- :sunrise: Add loki-logger plugin [#9399](https://github.com/apache/apisix/pull/9399)
+- :sunrise: Allow user configure DEFAULT_BUCKETS for prometheus plugin  [#9673](https://github.com/apache/apisix/pull/9673)
```
Comment (leslie-tsang) [inline CHANGELOG.md:87]:
```suggestion
- :sunrise: Allow user configure DEFAULT_BUCKETS for prometheus plugin [#9673](https://github.com/apache/apisix/pull/9673)
```

**Review by leslie-tsang — DISMISSED** [review]

**Review by monkeyDluffy6017 — DISMISSED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by kingluo — DISMISSED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -70,6 +71,38 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.4.0
+
+### Core
+
+- :sunrise: Support route-level MTLS [#9332](https://github.com/apache/apisix/pull/9332)
```
Comment (kayx23) [inline CHANGELOG.md:78]:
```suggestion
- :sunrise: Support route-level MTLS [#9322](https://github.com/apache/apisix/pull/9322)
```

**Comment by monkeyDluffy6017** [comment]
Could this pr be closed?

**Comment by AlinsRan** [comment]
> Could this pr be closed?

Why close? The master's change.log has not been updated yet, and this PR needs to be merged.

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by Neilblaze — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -70,6 +71,38 @@ title: Changelog
...
+### Core
+
+- :sunrise: Support route-level MTLS [#9322](https://github.com/apache/apisix/pull/9322)
+- :sunrise: Support id schema for global_rules [#9517](https://github.com/apache/apisix/pull/9517)
+- :sunrise: Support use a single long http connection to watch all resources for etcd [#9456](https://github.com/apache/apisix/pull/9456)
+- :sunrise: Support max len 256 for ssl label [#9301](https://github.com/apache/apisix/pull/9301)
+
+### Plugins
+
+- :sunrise: Support miltiple regex pattern matching for proxy_rewrite plugin [#9194](https://github.com/apache/apisix/pull/9194)
```
Comment (kayx23) [inline CHANGELOG.md:85]:
```suggestion
- :sunrise: Support multiple regex pattern matching for proxy_rewrite plugin [#9194](https://github.com/apache/apisix/pull/9194)
```

**Review by tao12345666333 — APPROVED** [review]

**Review by lingsamuel — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**